content/en/platform/corda/5.2/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -389,8 +396,8 @@ utxoLedgerService.query("DUMMY_CUSTOM_QUERY", Integer.class)
 Provide the name of the query (in this case `DUMMY_CUSTOM_QUERY`) and the return type. Since the result set is collected into an integer in the complex query example, use `Int` (or `Integer` in Java).
 Before executing, define the following:
 
-* How many results each page of the query should return (`setLimit`), default Int.MAX (2,147,483,647).
-* Define named parameters that are in the query and the actual value for them. {{< note >}} All parameters must be defined, otherwise the execution will fail. (`setParameter` or `setParameters`) {{</ note >}}
+* How many results each page of the query should return (`setLimit`), the default value is Int.MAX (2,147,483,647).
+* Define named parameters that are in the query and the actual value for them. ALl parameters must be defined, otherwise the execution will fail. (`setParameter` or `setParameters`).
 * Each state in the database has a timestamp value for when it was inserted. Set an * upper limit to only return states that were inserted before a given time. (`setTimestampLimit`)
 
 It is not necessary to call `ParameterizedQuery.setOffset` as the query pages the results based on each state's created timestamp.
```

Comment:
Sorry, I don't quite understand this sentence. Is there something missing?

Reply:
There was a line break missing. Maybe it's clearer now.

Reply:
Oh ok, "pages" is a verb. Got it now :)